experiments/mrmc_experiment/run_mrmc_experiment.py

```diff
@@ -424,6 +478,11 @@ def main(
     results_dir: Optional[str] = None,
     verbose: bool = False,
     dry_run: bool = False,
```

Comment:
We discussed waiting to change this until after the other run_experiment files were checked in, but since there were only a few comments, it is not hard to fix them now.